install/playbooks/roles/ldap/variables/main.yml

```diff
@@ -4,6 +4,4 @@
 users_defaults:
   uid_start: 1001
   gid_start: 1001
-  shell: /bin/dash
-  group: users
-  groups: mail
+  shell: /bin/bash
```

Comment:
TBH, I prefer to keep dash as the default shell, for security and resources usage. The shell is the default one under Debian. The purpose of the remote access is not to offer a fancy shell command line, but an last resort access, and more importantly, a mean to let some users access some files through SFTP, in their home directory. I will come back to you on this.

Reply:
However, we can add an optional "shell" field for each user, like this:
```
ldap:
...
  shell: '{{ user.shell | default(users_defaults.shell) }}'
```

Reply:
This works for me. I can add this.

What do you think of giving a `/usr/sbin/nologin` shell to the postmaster account? (Etiher directly in the ldap role or through a `ldap.postmaster.shell` variable in the same way.)